doc/python/morphology.rst

```diff
@@ -119,6 +119,7 @@ Cable cell morphology
         Radius of the cable (μm)
 
 .. class:: msegment
+Consider adding a sentence here explaining what this class does? (I don't know how to put comments in .rst file)
```

Comment:
This is just text and `.. class::` markup is extra. You should be able to just write any text here you like.